bee-message/src/constant.rs

```diff
@@ -6,4 +6,4 @@ pub const IOTA_SUPPLY: u64 = 2_779_530_283_277_761;
 
 /// Minimum amount of IOTA coins that need to be present in the smallest possible output in order not to be considered
 /// dust.
-pub const DUST_DEPOSIT_MIN: u64 = 1_000_000;
+pub const STORAGE_DEPOSIT_MIN: u64 = 1_000_000;
```

Comment:
```suggestion
pub const STORAGE_DEPOSIT_MIN: u64 = 1;
```
Please reduce it to 1, so we don't have problems during testing until this is not a const anymore